cardano-node/src/Cardano/Node/Tracing/Tracers/NodeToNode.hs

```diff
@@ -0,0 +1,762 @@
+              MsgClientDone)))
+        []
+        "Client termination message."
+  ]
+
+--------------------------------------------------------------------------------
+-- BlockFetchSerialised Tracer
+--------------------------------------------------------------------------------
+
+severityTBlockFetchSerialised :: BlockFetch.TraceLabelPeer peer
+  (TraceSendRecv (BlockFetch (Serialised blk) (Point blk))) -> SeverityS
+severityTBlockFetchSerialised (BlockFetch.TraceLabelPeer _ v) = severityTBlockFetch' v
+  where
+    severityTBlockFetch' (TraceSendMsg msg) = severityTBlockFetch'' msg
+    severityTBlockFetch' (TraceRecvMsg msg) = severityTBlockFetch'' msg
+
+    severityTBlockFetch'' (AnyMessageAndAgency _agency msg) = severityTBlockFetch''' msg
+
+    severityTBlockFetch''' :: Message (BlockFetch x (Point blk)) from to
+                                   -> SeverityS
+    severityTBlockFetch''' MsgRequestRange {} = Info
+    severityTBlockFetch''' MsgStartBatch {}   = Info
+    severityTBlockFetch''' MsgNoBlocks {}     = Info
+    severityTBlockFetch''' MsgBlock {}        = Info
+    severityTBlockFetch''' MsgBatchDone {}    = Info
+    severityTBlockFetch''' MsgClientDone {}   = Info
+
+namesForTBlockFetchSerialised :: BlockFetch.TraceLabelPeer peer
+  (TraceSendRecv (BlockFetch (Serialised blk) (Point blk))) -> [Text]
+namesForTBlockFetchSerialised (BlockFetch.TraceLabelPeer _ v) =
+  "NodeToNode" : namesTBlockFetch v
+  where
+    namesTBlockFetch (TraceSendMsg msg) = "Send" : namesTBlockFetch' msg
+    namesTBlockFetch (TraceRecvMsg msg) = "Recieve" : namesTBlockFetch' msg
+
+    namesTBlockFetch' (AnyMessageAndAgency _agency msg) = namesTBlockFetch'' msg
+
+    namesTBlockFetch'' :: Message (BlockFetch x (Point blk)) from to
+                               -> [Text]
+    namesTBlockFetch'' MsgRequestRange {} = ["RequestRange"]
+    namesTBlockFetch'' MsgStartBatch {}   = ["StartBatch"]
+    namesTBlockFetch'' MsgNoBlocks {}     = ["NoBlocks"]
+    namesTBlockFetch'' MsgBlock {}        = ["Block"]
+    namesTBlockFetch'' MsgBatchDone {}    = ["BatchDone"]
+    namesTBlockFetch'' MsgClientDone {}   = ["ClientDone"]
+
+-- TODO Tracers
+instance ( ConvertTxId blk
+         , HasTxId (GenTx blk)
+         , ConvertRawHash blk
+         , HasTxs blk
+         )
+      => LogFormatting (AnyMessageAndAgency (BlockFetch (Serialised blk) (Point blk))) where
+  forMachine DMinimal (AnyMessageAndAgency stok (MsgBlock _blk)) =
+    mkObject [ "kind" .= String "MsgBlock"
+             , "agency" .= String (pack $ show stok)
+            -- , "blockHash" .= renderHeaderHash (Proxy @blk) (blockHash blk)
+            -- , "blockSize" .= toJSON (estimateBlockSize (getHeader blk))
+             ]
+
+  forMachine _dtal (AnyMessageAndAgency stok (MsgBlock _blk)) =
+    mkObject [ "kind" .= String "MsgBlock"
+             , "agency" .= String (pack $ show stok)
+          -- TODO
+          -- , "blockHash" .= renderHeaderHash (Proxy @blk) (blockHash blk)
```

Comment:
TODO in this PR?

Reply:
So, it turns out that the legacy setup doesn't have a complete coverage for the structured tracing -- in particular, the serialised-exchange tracers for `BlockFetch` and `ChainSync` don't have `ToObject` and rely on `Show` instead.

It is out of scope for us now to deal with that -- so we'll just follow the status quo.

@jutaro says we'll have a version with a `Show` stub early tomorrow.

The `TODO` will stay though, since it's a todo for the legacy as well -- to be rectified at some point later.

Reply:
Added the forHuman implementation, will work later on the complete forMachine implementation.